.github/workflows/docker.yml

```diff
@@ -15,8 +15,8 @@ jobs:
       matrix:
         runtime:
           - name: gtk-latest
```

Comment:
I see how `gtk-latest` is confusing and seeing that builds from `main` will pretty much never succeed, I think we can just remove the `gtk-latest` and move the changed configuration into `gtk-4.6`.